views/index.html

```diff
@@ -79,7 +79,7 @@ <h1 id="header-title" class="col-md-4 lane centered-text">Phoenix Timeline</h1>
 
                 <!-- SLIDER -->
                 <div class="ui-control col-md-4">
-                    <input type="range" id="scale-slider" min=1 max=13 value=5 step=.1 oninput="update()">
+                    <input type="range" id="scale-slider" min=1 max=8 value=5 step=.1 oninput="update()">
```

Comment:
13 was picked as a maximum range slider value because it shows a week almost exactly (13^2 = 169 hours vs 24 * 7 = 168 hours). Is there a reason for the change to 8?